app/travel_processor/karma/cap-server.js

```diff
@@ -62,9 +65,9 @@ async function node() {
     const started = data.match(/server listening on {.*url:.*'(?<url>.+)'.*}/);
     if (started) return new URL(started.groups.url);
   };
-  const serverUrl = await spawnServer("npm", ["start"], "../..", isReady);
+  const [serverUrl, pid] = await spawnServer("npm", ["start"], "../..", isReady);
```

Comment:
the same changes need to be done in the java() function above, too (PR voting runs the same tests on both stacks, Java and Node.js)

Reply:
Thank for the tip, was too focused on node. Added the change with a new commit.